src/Microsoft.DocAsCode.Build.Engine/DocumentBuildContext.cs

```diff
@@ -86,6 +87,8 @@ public DocumentBuildContext(string buildOutputFolder, IEnumerable<FileAndType> a
 
         public ImmutableArray<string> XRefMapUrls { get; }
 
+        public ImmutableArray<string> XRefServiceUrls { get; }
```

Comment:
if so what about XRefMapUrls?

---
In reply to: [126865746](https://github.com/dotnet/docfx/pull/1837#discussion_r126865746) [](ancestors = 126865746)